src/extension/settingsHelper.ts

```diff
@@ -63,4 +67,22 @@ export class SettingsHelper {
                 return settingsJson["typescript.tsdk"] || "";
             });
     }
+
+    /**
+     * Reads and parses the launch.json file.
+     */
+    public static readLaunchJson(): Q.Promise<any> {
+        let fileSystem = new FileSystem();
+        let launchJsonPath = SettingsHelper.launchJsonPath;
+
+        return fileSystem.exists(launchJsonPath)
```

Comment:
Since we are aiming at returning a JSON object, I believe returning `null` if the file is not there is a good contract for consumers of this method.